understanding/20/keyboard.html

```diff
@@ -115,8 +141,12 @@ <h2>Examples of Keyboard</h2>
          <dd>A PDA device that is usually operated via a stylus has an optional keyboard that can
                be attached.  The keyboard allows full Web browsing in standard fashion.  The Web
                content is operable because it was designed to work with keyboard-only access.</dd>
+        <dt>Example 7: Simple search form with pointer-operable submit button</dt>
+        <dd>A search form includes a text input field followed by a submit button. The submit button itself
+               has been coded so that it does <em>not</em> receive focus, and can only be activated using a pointer input.
+               However, since keyboard users can submit the search by pressing <kbd>Enter</kbd> in the text input
+               after typing their search terms, the form passes this Success Criterion.</dd>
```

Comment:
```suggestion
               after typing their search terms, the form satisfies this Success Criterion.</dd>
```

Reply:
we have a few other cases in the spec where we use the phrase "passes this Success Criterion", so leaving this as is. however, edited the ones that originally said "passes the requirements", as there it's consistently "satisfies the requirements"